src/panels/LogPanel/ScriptLogEntry.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import { ActionIcon, Code, Group, Text, Tooltip } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { CopyToClipboardButton } from '@/components/CopyToClipboardButton/CopyToClipboardButton';
+import { RerunScriptIcon } from '@/icons/icons';
+
+interface Props {
+  script: string;
+}
+
+export function ScriptLogEntry({ script }: Props) {
+  const luaApi = useOpenSpaceApi();
+  const [expanded, setExpanded] = useState(false);
+
+  function runScript() {
+    luaApi?.scheduleScript(script, 0);
+  }
+
+  return (
+    <Group wrap={'nowrap'} justify={'space-between'} align={'start'}>
+      <Code color={"dark.7"} w={'100%'}>
+        <Text
+          truncate={expanded ? undefined : 'end'}
+          onClick={() => setExpanded(!expanded)}
+          style={{ cursor: 'pointer', overflowWrap: 'anywhere' }}
+        >
+          {script}
+        </Text>
+      </Code>
+      <Group wrap={'nowrap'} gap={'xs'}>
+        <CopyToClipboardButton value={script} />
+        <Tooltip label={'Rerun script'}>
+          <ActionIcon
+            onClick={runScript}
+            variant={'light'}
+            color={'green'}
+            aria-label={'Rerun script'}
```

Comment:
Nice with the aria label. However, do we also need some type of "visually hidden" text that says what the script actually is? Now I don't think a screen reader will get this information

Reply:
It might also be confusing for screen reader users now since the button for each line says exactly the same thing in the aria label. 

Worth some consideration